exercises/concept/cache/.meta/example.go

```diff
@@ -0,0 +1,36 @@
+package cache
+
+func GetValueFromFastestLayer(cacheLayers []Cache, key string) (string, error) {
+	ch := make(chan string, len(cacheLayers))
+	for _, cache := range cacheLayers {
+		go func(cache Cache) {
+			value, _ := cache.Get(key)
+			ch <- value
+		}(cache)
+	}
+
+	for range cacheLayers {
+		value := <-ch
+		if value != "" {
+			return value, nil
+		}
+	}
+
+	return "", ErrKeyNotExist
+}
+
+func SetValueToAllLayers(cacheLayers []Cache, key string, value string) error {
+	ch := make(chan error, len(cacheLayers))
+	for _, cache := range cacheLayers {
+		go func(cache Cache) {
+			err := cache.Set(key, value)
+			ch <- err
+		}(cache)
+	}
+
+	for range cacheLayers {
+		<-ch
+	}
+
+	return nil
+}
```

Comment:
A couple of thoughts about this task (and the solution):
* Currently, the function always returns nil so it could also just not have a return value at all.
* The instructions should clarify what happens in case Set returns an error.
* Overall, the required solution here is very similar to the one of the first task. Usually we try to bring up some new aspect in each task. Have another look at the learning objectives, maybe the task can be adjusted/re-written to cover something different. For example, currently your tasks don't cover ranging over a channel at all.